packages/composer/amazeelabs/graphql_directives/README.md

```diff
@@ -103,6 +127,23 @@ type Query {
 }
 ```
 
+### `@type`
+
+Annotate an object type with a specific id that will be used for interface- and
+union type resolution.
+
+```graphql
+union Letters @prop(key: "type") = A | B
+
+type A @type(id: "a") {
+  type: String!
+}
+
+type B @type(id: "B") {
```

Comment:
Should it be `id: "b"` ? (lowercase)

Reply:
> How does it replace
> 
> ```
> @resolveProperty(path: "field_text.0.processed")
> @resolveProperty(path: "field_entity_reference.entity")
> ```
> 
> ?

Not at all. My plan is to move `@resolveProperty` (and most other directives we have) to plugins to make them more maintainable.